src/main/java/org/acra/config/ConfigurationBuilder.java

```diff
@@ -742,14 +742,6 @@ int connectionTimeout() {
     }
 
     @NonNull
-    ReportField[] customReportContent() {
```

Comment:
It is not public, so only uses inside the package have to be checked. The usage was inside ACRAConfiguration, but this was replaced with reportContent while moving the logic. (also Android studio didn't find any other usage)